sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusMessage.cs

```diff
@@ -216,7 +225,7 @@ public TimeSpan TimeToLive
             }
             set
             {
-                Argument.AssertPositive(value, nameof(TimeToLive));
+                Argument.AssertPositive(value, nameof(value));
```

Comment:
According to Framework Design Guidelines, we should use `value` rather than the property name for ArgumentExceptions thrown from setters.

Reply:
![](https://media3.giphy.com/media/ALalVMOVR8Qw/giphy.gif)